File: uri/python/1074.py
value = int(input())
inputs = []

for x in range(value):
    inputs.append(int(input()))

for i in inputs:
    if i == 0:
        print('NULL')
    elif i % 2 == 0 and i > 0:
        print('EVEN POSITIVE')
    elif i % 2 == 0 and i < 0:
        print('EVEN NEGATIVE')
    elif i % 2 != 0 and i > 0:
        print('ODD POSITIVE')
    elif i % 2 != 0 and i < 0:
        print('ODD NEGATIVE')

File: uri/c/1074.c
#include <stdio.h>

int main() {

    int N, X;
    scanf("%d", &N);

    while (N > 0) {

        scanf("%d", &X);

        if (X % 2 == 0 && X > 0) {
            printf("EVEN POSITIVE\n");
        } else if (X % 2 == 0 && X < 0) {
            printf("EVEN NEGATIVE\n");
        } else if (X % 2 != 0 && X > 0) {
            printf("ODD POSITIVE\n");
        } else if (X % 2 != 0 && X < 0) {
            printf("ODD NEGATIVE\n");
        } else if (X == 0) {
            printf("NULL\n");
        }

        N--;
    }

    return 0;

}